### PR TITLE
Update package.py for vacuumms 1.2.0

### PR DESCRIPTION
**Pull request by frankwillmore**

### CODE REVIEW

**Comment by frankwillmore** [comment]
Update of spack package for new VACUUMMS version (1.2.0). Also, clarified license is MIT.

File: var/spack/repos/builtin/packages/vacuumms/package.py [hunk 1]
```diff
@@ -16,25 +16,48 @@ class Vacuumms(CMakePackage):
...
+        "1.1.4",
+        sha256="13052bafca8622df5f14613b3d4cddd50bae725e0c13c1df7f3cdee28a2e673a",
+        deprecated=False,
+    )
+    version(
+        "1.2.0",
+        sha256="1836e6c29d1e83a79f9c6a9b686d1419bd72618aba7c46d3d038373808519d56",
+        preferred=True,
+        deprecated=False,
+    )
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/vacuumms/package.py]: The packaging convention is to order version directives from newest to oldest so please reverse the ordering of these.  (See the first note in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching.)

File: var/spack/repos/builtin/packages/vacuumms/package.py [hunk 2]
```diff
@@ -16,25 +16,48 @@ class Vacuumms(CMakePackage):
...
 
+    variant("test", default=True, description="enable CMake testing")
     variant("tiff", default=False, description="Build TIFF utilities")
     variant("cuda", default=False, description="Build CUDA applications and utilities")
+    variant("variational", default=False, description="Build VARIATIONAL module")
+    variant("voronoi", default=False, description="Build VORONOI applications and utilities")
+    variant(
+        "VOROPP_HOME",
+        default="/opt/voropp",
+        description="""voro++ location""",
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/vacuumms/package.py]: Curious.  Why have extra double quotes for the description?

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed the branch and all (affected) version sha256.  Just a couple of minor changes.

File: var/spack/repos/builtin/packages/vacuumms/package.py (hunk 1 above)
Comment (frankwillmore) [reply var/spack/repos/builtin/packages/vacuumms/package.py]: done

**Review by tldahlgren — APPROVED** [review]
Thanks.  And I'm glad you knew what I meant about the *extra* double quotes.

**Pull request merged**